NewApp/ngsa-csharp/Ngsa.DataService/Core/CommandLine.cs

```diff
@@ -91,6 +91,8 @@ public static async Task<int> RunApp(string secretsVolume, LogLevel logLevel, bo
         {
             try
             {
+                Task art = DisplayAsciiArt();
```

Comment:
make ascii art async for animation

Reply:
TODO - I was playing around with this for fun - probably need to remove or limit the animation. It is kind of cool :)

Maybe put a "Press any key" message and animate until the key is pressed?

ASCII art only appears on --help and --dry-run. It would totally hose our fluent bit logs